stores/governance/src/main/java/com/bloxbean/cardano/yaci/store/governance/storage/impl/CommitteeMemberStorageImpl.java

```diff
@@ -74,13 +89,14 @@ public List<CommitteeMemberDetails> getActiveCommitteeMembersDetailsByEpoch(int
                                                                 .and(COMMITTEE_DEREGISTRATION.CERT_INDEX.gt(COMMITTEE_REGISTRATION.CERT_INDEX))
```

Comment:
Do you need to also check against tx_index and then cert_index? 
Looks like we don't have tx_index field in committee_registraion and committee_deregistration tables. I think we need to add that column to find the correct entry. You may want to refer to staking related tables.